src/panels/lovelace/editor/config-elements/hui-stack-card-editor.ts

```diff
@@ -107,26 +107,31 @@ export class HuiStackCardEditor extends LitElement
                         : "ui.panel.lovelace.editor.edit_card.show_visual_editor"
                     )}
                   </mwc-button>
-                  <ha-icon-button
-                    id="move-before"
-                    title="Move card before"
-                    icon="hass:arrow-left"
+
+                  <mwc-icon-button
                     .disabled=${selected === 0}
+                    label="Move card before"
```

Comment:
While at it, can we localize these?

Reply:
I was just going to ask should I do it 🙂
Sure, give me a second.

Reply:
@bramkragten last question.
In en.json we have card written with first letter [upper ](https://github.com/home-assistant/frontend/blob/dev/src/translations/en.json#L2246)and [lower](https://github.com/home-assistant/frontend/blob/dev/src/translations/en.json#L2238).
I think this should be consistent.

Reply:
It probably should, but in a sentence, it should definitely not be uppercase. In a short action format... yeah probably also not?

Reply:
Thanks for your clarification.
I'll try to optimize the `title` property in the next PR's. In many places, it is used without dot notation and `hass` is used without the exclamation point: https://github.com/home-assistant/frontend/blob/dev/src/panels/config/devices/ha-config-device-page.ts#L268

Reply:
it should be without exclamation

Reply:
there are about 380 places that use an exclamation. Like here (https://github.com/home-assistant/frontend/blob/dev/src/panels/config/automation/condition/types/ha-automation-condition-time.ts): 
![image](https://user-images.githubusercontent.com/1741838/95199706-88401500-07dd-11eb-8a7e-a95181f6e01f.png)
In the same class has is defined as:
`@property({ attribute: false }) public hass!: HomeAssistant;`

So all those places should be changed? I'm confused 🤷‍♂️

Reply:
Do you know how typescript works and what the exclamation means?

Reply:
But if `hass` is defined as with a `!` then it is redundant in the rest of the file. Not that it matters for the actual output.

Reply:
@bramkragten I just checked to be sure that my thinking was correct. Nothing changed, it is still a not-null assertion operator (https://stackoverflow.com/questions/42273853/in-typescript-what-is-the-exclamation-mark-bang-operator-when-dereferenci)
So when `hass` is defined with `!` then `!` is redundant in labels and titles.
I'll pick a couple of files and will do a PR with the changes and we can discuss the change there.